supabase/migrations/20260713114104_harden_rbac_compat_cleanup_after_rls.sql

```
@@ -5364,6 +5364,20 @@ ALTER FUNCTION public.check_if_org_can_exist() OWNER TO postgres;
 REVOKE ALL ON FUNCTION public.check_if_org_can_exist() FROM PUBLIC;
 GRANT EXECUTE ON FUNCTION public.check_if_org_can_exist() TO service_role;
 
+DROP TRIGGER IF EXISTS check_privileges ON public.org_users;
+CREATE TRIGGER check_privileges
+BEFORE INSERT OR UPDATE OF user_id, org_id, rbac_role_name ON public.org_users
+FOR EACH ROW
+WHEN (
+  current_setting('request.jwt.claim.role', true) = 'authenticated'
+  AND NOT (
+    current_setting('request.jwt.claim.email', true) = ANY (
```

SQLFluff (4.2.2)

[error] Function name not immediately followed by parenthesis.

(LT06)